src/_gettsim/erziehungsgeld/erziehungsgeld.py

```diff
@@ -2,14 +2,82 @@
 
 from __future__ import annotations
 
+from dataclasses import dataclass
+from typing import Any
+
 from ttsim import (
     AggType,
     RoundingSpec,
     agg_by_p_id_function,
+    param_function,
     policy_function,
 )
 
 
+@dataclass(frozen=True)
+class EinkommensgrenzeErziehungsgeldGegebenFamilienstandNachSatzArt:
+    regelsatz: float
+    budgetsatz: float
+
+
+@dataclass(frozen=True)
+class EinkommensgrenzeErziehungsgeldNachFamilienstand:
+    alleinerziehend: EinkommensgrenzeErziehungsgeldGegebenFamilienstandNachSatzArt
+    paar: EinkommensgrenzeErziehungsgeldGegebenFamilienstandNachSatzArt
+
+
+@dataclass(frozen=True)
+class EinkommensgrenzeErziehungsgeld:
+    regulär: EinkommensgrenzeErziehungsgeldNachFamilienstand
+    reduziert: EinkommensgrenzeErziehungsgeldNachFamilienstand
+    maximalalter_reguläres_limit_monate: int
```

Comment:
Why not keep this as a scalar parameter? Is often used without the other components and it is not an income threshold.